kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/auth/AuthLocalData.java

```diff
@@ -85,34 +90,34 @@ public void run() {
     }
 
     @Override
-    public void getAuthToken(@NonNull final Callback<AuthToken> callback) {
-        Runnable command = new Runnable() {
+    public void getAppId(@NonNull final Callback<String> callback) {
+        final Runnable command = new Runnable() {
```

Comment:
do we need to save weak reference to this callback if appID is not known yet and call it setAuthToken. once call back is called remove reference

Reply:
no this is coming from the sharedpref and returns immediately ca response.